apollo/interfaces/generic/main.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import os
 from datetime import datetime, timezone, timedelta
+from time import sleep
```

Comment:
☝️ unused?

Reply:
good catch @97nitt , yes, added that for testing, thanks